app/src/androidTest/java/com/android/sample/ui/profile/ProfileScreenTests.kt

```diff
@@ -223,7 +223,7 @@ class ProfileScreenTest {
           clear() // Clears all fields to start fresh and avoid unwanted hour/minute/second
           set(Calendar.YEAR, 2024) // Set year to 2024
           set(Calendar.MONTH, Calendar.DECEMBER) // Set month to December
-          set(Calendar.DAY_OF_MONTH, 5) // Set day to 5
+          set(Calendar.DAY_OF_MONTH, 6) // Set day to 5
```

Comment:
This test has an issue that will be solved by the person creating it. Basically, it blocks my CI because we need to update the number to the current day each day. For my PR to pass and since this will be solved in the coming days, I have just commented the test.

Reply:
Yes definitely. @hmaryam84 shall take care of it 👍